custom_components/versatile_thermostat/prop_algorithm.py

```diff
@@ -161,6 +163,15 @@ def _calculate_internal(self):
             )
             self._on_percent = self._calculated_on_percent
 
+        if self._max_on_percent is not None and self._on_percent > self._max_on_percent:
+            _LOGGER.debug(
+                "%s - Heating period clamped to %s%% (instead of %s%%) due to max_on_percent setting.",
```

Comment:
on_percent is not a percentage (yes I know ...). Juste remove the % sign.

Reply:
Thanks for the review! 

> This is fine. Just a typo.

I have already fixed the "%" typos. 

> , and please update the expert_mode in README (I will report after in README-fr).

Currently the expert mode section is related to "thermostat over climate" section, which does not fit the new setting. My suggestion is to put the info near the end of the configuration section after "Control of a central boiler" and before "Parameter summary". What do you think about this? 

> You will have to update the strings.json and en.json (I will do the Fr)

I am blanking a bit on this. My understanding is that `strings.json` holds frontend "text" that need to be translated. My change does not have any frontend exposure, expect of the attributes which can be visible in the debug menu of HA, but I don't think these should be translated. Can you please point me in the right direction on this?

Reply:
> Currently the expert mode section is related to "thermostat over climate" section, which does not fit the new setting. My suggestion is to put the info near the end of the configuration section after "Control of a central boiler" and before "Parameter summary". What do you think about this?

You are right. That is fine. And if you can move the over_climate expert § into this chapter. If you want, I can do it.

>  My understanding is that strings.json holds frontend "text" that need to be translated. My change does not have any frontend exposure

Once more you are totally right. Forget this comment.